### PR TITLE
Partiton: Improved ks_pre partition removal

### PR DESCRIPTION
**Pull request by stdweird**
Fixes https://github.com/quattor/aii/issues/115 and #50

### CODE REVIEW

**Comment by jrha** [comment]
LGTM

File: src/main/perl/Blockdevices.pm [hunk 1]
```diff
@@ -104,6 +104,19 @@ sub should_create_ks
 	$this_app->error ("should_create_ks method not defined for this class");
 }
 
+# Return the size of metadata to wipe in MB
+sub get_clear_mb
+{
+    my $self = shift;
+
+    # Clear two times the alignment, but at least 1MB
+    # TODO: in what unit is/was the aligment? assuming in bytes because the align_sect does a /512 (5112b in a sect)
```
Comment (jrha) [inline src/main/perl/Blockdevices.pm]: How many bytes in a sector?

**Comment by jrha** [comment]
poke @stdweird?

**Comment by stdweird** [comment]
@jrha sorry, missed that somehow. fixed now

**Comment by jrha** [comment]
:smile: LGTM

**Pull request merged**